🥋 Update docs with chaining emit matchers (#693)
TrueFiEng/Waffle: docs/source/matchers.rst

    .to.emit(contract, 'AddAddress')
    .withArgs(hashedStreet, hashedCity);

> If you are using the latest version of Waffle, you can't chain :code:`emit` matchers like in example belove.

Reviewer: ```suggestion
If you are using Waffle version :code:`3.4.4` or lower, you can't chain :code:`emit` matchers like in example below.
```

.. code-block:: ts

  await expect(tx)
    .to.emit(contract, 'One')
    .to.emit(contract, 'Two');

> But this feature is available in alpha version of Waffle.

Reviewer: ```suggestion
This feature will be available in Waffle version 4.
```
